src/sync.test.js

```diff
@@ -65,6 +65,24 @@ pluginTester({
         });
       });
       `
+    },
+    'Does not count commented out expect statements': {
```

Comment:
Can you add a comment like:

```js
const a = 1; // expect(a).toBe(1)
const b = 1; /* expect (b).toBe(1) */
```

Reply:
You mean just to add this code into the test code? why?

Reply:
Yeah in the body of the code being transpiled, to make sure lines of code with trailing comments are not counted too

Reply:
Done.